stage0/interface.go

```diff
@@ -59,3 +59,7 @@ func getStage1InterfaceVersion(cdir string) (int, error) {
 func interfaceVersionSupportsHostname(version int) bool {
 	return version > 1
 }
+
+func interfaceVersionSupportsStop(version int) bool {
+	return version > 2
+}
```

Comment:
I think this function could be deleted as discussed in https://github.com/coreos/rkt/pull/2438#discussion-diff-65917341